lua/telescope/builtin/__internal.lua

```diff
@@ -993,6 +993,19 @@ internal.colorscheme = function(opts)
     end, vim.fn.getcompletion("", "color"))
   )
 
+  if opts.ignore_builtins then
+    -- stylua: ignore
+    local builtins = {
+      "zellner", "torte", "slate", "shine", "ron", "quiet", "peachpuff",
+      "pablo", "murphy", "lunaperche", "koehler", "industry", "evening",
+      "elflord", "desert", "delek", "darkblue", "blue", "morning", "vim",
+      "habamax", "retrobox", "sorbet", "zaibatsu", "wildcharm", "default",
```

Comment:
not that happy about that implementation, but ig we can do that.

But can we at least sort them alphabetically. thanks

Reply:
yeah, I couldn't think of any other reliable method of getting all builtin, but if you suggestions, I'm all ears.

Otherwise, I sorted them alphabetically.